CONTRIBUTING.md

```diff
@@ -98,5 +98,148 @@ bug reports or feature requests. Before doing so however, please take into consi
 * When submitting a feature request:
     * Explain why you need the feature &mdash; what's your use-case, what's your domain. Explaining the problem you face is more important than suggesting a solution.
       Report your problem even if you don't have any proposed solution. If there is an alternative way to do what you need, then show the code of the alternative.
+
+
+# How to contribute
+
+Before we get started, thank you for considering contributing to Ktor. It's awesome of you!
+
+There are multiple ways you can contribute:
+
+* Code
+* Documentation
+* Community Support
+* Feedback/Issue reports
+
+Independently of how you'd like to contribute, please make sure you read and comply with the [Code of Conduct](CODE_OF_CONDUCT.md).
+
+## Code
+
+There are many bugs and features in the Ktor backlog and you're free to pick any of them. We do recommend however starting
+with some of the [low hanging fruit](https://youtrack.jetbrains.com/issues?q=%23Ktor%20%20%20%23%7BUp%20For%20Grabs%7D%20%20%23Unresolved%20).
+
+### Pull Requests
+
+Contributions are made using Github [pull requests](https://help.github.com/en/articles/about-pull-requests):
+
+1. Fork the Ktor repository and work on your fork.
```

Comment:
This looks like a duplicate of the top part. Should we drop it at the top(line 23)?